api/TrustedHTML.json

```
@@ -14,7 +14,8 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": "147"
+            "version_added": false,
+            "impl_url": "https://bugzil.la/1994690"
```

Verified that this is in Nightly
```suggestion
            "version_added": "preview"
```